packages/server/src/server.ts

```diff
@@ -94,7 +96,6 @@ export const createApp = async (options: CreateAppOptions): Promise<{
 						userAgent = userAgent.join(", ");
 					}
 					logger.info("Failed login attempt",
-						field("password", cookies.password),
```

Comment:
I think it'd be ok to log the hash of the password versus the expected hash. See #952

Reply:
It is ok, but it wouldn't add any information other than "it doesn't match". Therefor it will only make the logs unreadable.